src/component/Nav/component/Modal.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Modal.scss';
+
+const Modal = ({ open, close }) => {
+  return (
+    <div>
+      <div className={open ? 'openModal modal' : 'modal'}>
+        {open ? (
+          <div className="modalContent" onMouseLeave={() => close()}>
```

Comment:
```suggestion
          <div className="modalContent" onMouseLeave={close}>
```
* 인자를 전달할 목적이 아니라면, 함수 자체를 할당할 수 있습니다.